src/kits/communityWebServer/StaticFileWeblet.sedona

```diff
@@ -0,0 +1,259 @@
+//
+// Copyright (c) 2018
+// Licensed under the Academic Free License version 3.0
+//
+// History:
+//   30 Jun 18  Vincent Wang  Static File Weblet
+//
+
+** 
+** StaticFileWeblet provides the ability to serve static files(html, css, js,
+** images etc) through http, so that user can use sedona as a minimal web
+** server and host a static web app just in sedona, without any outside
+** dependency. 
+** to save space on sedona, gzipped file is supported and I recommend to use it
+** since that will be more efficient(less bytes to be loaded and transferred)
+** 
+class StaticFileWeblet extends Weblet
+{
+////////////////////////////////////////////////////////////////
+// Weblet Registration
+////////////////////////////////////////////////////////////////
+
+  static void _sInit()
+  {
+    instance.register()
```

Comment:
We should consider adding a config property to enable/ disable any weblet after registering it. For example, spy and spyREST should be disabled for deployments. Similarly, one should allow users to disable this weblet, if required.

This would need changes to WebService and Weblet sedona classes and will also help REST service later.

Reply:
agreed. since this is general enough for weblet, I just created a new issue #26 for it, will work on it later.

Reply:
LGTM